src/saving_tools.jl

```diff
@@ -208,6 +209,7 @@ function tag!(d::AbstractDict{K,T};
     # Get the appropriate keys
     commitname = keyname(d, :gitcommit)
     patchname = keyname(d, :gitpatch)
+    message_name = keyname(d, :gitmessage) ### added
```

Comment:
What information does the `### added` comment provide? Does it really help? Remove the comment.